features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/RumMonitor.kt

```diff
@@ -323,6 +323,15 @@ interface RumMonitor {
      */
     fun stopSession()
 
+    /**
+     * Adds view loading time to current RUM view based on the time elapsed since the view was started.
+     * This method should be called only once per view.
```

Comment:
Why do we enforce it requirement on the client side? Cannot we just check in the implementation that if it was called already for the view, then ignore subsequent calls?

Reply:
we follow same approach as on iOS and for now on their end they allow it to be called multiple times. @ganeshnj maybe you want to add some thoughts here ?

Reply:
> [!IMPORTANT]
> I agree with @0xnm , we can't ask the customer to know whether or not the method has already been called. They could have several code paths that would lead to calling this method, and it would be a hassle to ask them to track whether or not that method have been called for a given View. 
> We need to make our SDK resilient enough to handle "misuse".